xapian-core/weight/tfidfweight.cc

```diff
@@ -210,6 +213,14 @@ TfIdfWeight::get_wdfn(Xapian::termcount wdf, Xapian::termcount doclen,
 	    double den = 1 + log(wdf_avg);
 	    return num / den;
 	}
+	case 'a': {
+	    if (wdf == 0) return 0;
+	    return (0.2 + 0.8 * log(1.0 + wdf));
+	}
+	case 'S': {
+	    if (wdf == 0) return 0;
+	    return (sqrt(wdf - 0.5) + 1);
+	}
```

Comment:
Where's "S" defined?  I don't see it listed at http://people.csail.mit.edu/jrennie/ecoc-svm/smart.html or https://nlp.stanford.edu/IR-book/html/htmledition/document-and-query-weighting-schemes-1.html